plugins/modules/cloud/misc/proxmox_disk.py

```diff
@@ -0,0 +1,688 @@
+        the value is added to the actual size of the volume
+        and without it, the value is taken as an absolute one.
+    type: str
+  bwlimit:
+    description:
+      - Override I/O bandwidth limit (in KB/s).
+      - Used only when I(state=moved).
+    type: int
+  delete_moved:
+    description:
+      - Delete the original disk after successful copy.
+      - By default the original disk is kept as unused disk.
+      - Used only when I(state=moved).
+    type: bool
+  target_disk:
+    description:
+      - The config key the disk will be moved to on the target VM (for example, C(ide0) or C(scsi1)).
+      - Default is the source disk key.
+      - Used only when I(state=moved).
+    type: str
+  target_storage:
+    description:
+      - Target storage.
+      - Used only when I(state=moved).
+    type: str
+  target_vmid:
+    description:
+      - The (unique) ID of the VM.
+      - Used only when I(state=moved).
+    type: int
+  timeout:
+    description:
+      - Timeout to wait when moving disk.
+      - Used only when I(state=moved).
+    type: int
+    default: 600
+  aio:
+    description:
+      - AIO type to use.
+    type: str
+    choices: ['native', 'threads', 'io_uring']
+  backup:
+    description:
+      - Whether the drive should be included when making backups.
+    type: bool
+  bps:
+    description:
+      - Maximum r/w speed in bytes per second.
+    type: int
+  bps_max_length:
+    description:
+      - Maximum length of I/O bursts in seconds.
+    type: int
+  bps_rd:
+    description:
+      - Maximum read speed in bytes per second.
+    type: int
+  bps_rd_max_length:
+    description:
+      - Maximum length of read I/O bursts in seconds.
+    type: int
+  bps_wr:
+    description:
+      - Maximum write speed in bytes per second.
+    type: int
+  bps_wr_max_length:
+    description:
+      - Maximum length of write I/O bursts in seconds.
+    type: int
+  cache:
+    description:
+      - The drive's cache mode.
+    type: str
+    choices: ['none', 'writethrough', 'writeback', 'unsafe', 'directsync']
+  cyls:
+    description:
+      - Force the drive's physical geometry to have a specific cylinder count.
+    type: int
+  detect_zeroes:
+    description:
+      - Controls whether to detect and try to optimize writes of zeroes.
+    type: bool
+  discard:
+    description:
+      - Controls whether to pass discard/trim requests to the underlying storage.
+    type: str
+    choices: ['ignore', 'on']
+  format:
+    description:
+      - The drive's backing file's data format.
+    type: str
+    choices: ['raw', 'cow', 'qcow', 'qed', 'qcow2', 'vmdk', 'cloop']
+  heads:
+    description:
+      - Force the drive's physical geometry to have a specific head count.
+    type: int
+  import_from:
+    description:
+      - Import volume from this existing one. To use this parameter you have to specify C(size=0) when creating disk.
```

Comment:
Please ensure all references to parameter names (with or without values) use the `I(...)` markup.
```suggestion
      - Import volume from this existing one. To use this parameter you have to specify I(size=0) when creating disk.
```